features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,33 @@
+
+When("I visit the {string} page") do |page_path|
+    if page_path == 'sign up'
+        visit new_user_registration_path
+    else
+        return false
+    end
+end
+
+Then("I should be able to fill in {string} with {string}") do |input_field, input_value|
+    fill_in input_field, with: input_value
+end
```

Comment:
This is the exact same step definition as the one in the bottom. The feature steps should be refactored so that you only need to use one step definition

Reply:
Removed and refactored!